actionview/lib/action_view/dependency_tracker/wildcard_resolver.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module ActionView
+  class DependencyTracker # :nodoc:
+    class WildcardResolver # :nodoc:
+      def initialize(view_paths, dependencies)
+        @view_paths = view_paths
+
+        @wildcard_dependencies, @explicit_dependencies =
+          dependencies.partition { |dependency| dependency.end_with?("/*") }
```

Comment:
Does this mean we still only support the wildcards/interpolations when they're trailing? I think that's an okay limitation to have (maybe beneficial even to avoid matching too many/every template) but let's be sure to document that if so.

Reply:
Yes, I just leveraged the existing code for explicit dependencies that let people do `Template Dependency: events/*`. I wanted to keep it simple for an initial implementation, and I definitely agree that wildcards in other positions could be a really easy performance footgun.

I added a new example to the `CacheHelper` docs to mention this new behavior, happy to make it more detailed if you think it would be beneficial!